packages/fxa-auth-server/lib/routes/subscriptions/paypal.ts

```diff
@@ -26,6 +27,13 @@ export class PayPalHandler extends StripeHandler {
     super(log, db, config, customs, push, mailer, profile, stripeHelper);
     this.paypalHelper = Container.get(PayPalHelper);
   }
+
+  /**
+   * Get PayPal Checkout Token.
+   */
+  async getCheckoutToken() {
+    return await this.paypalHelper.getCheckoutToken();
```

Comment:
There's a few calls that are pretty standard on every http call:
https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/lib/routes/subscriptions/stripe.ts#L178

The minimum is generally the log statement, and the customs check (prevents abuse of our endpoints).